src/components/Footer/index.tsx

```diff
@@ -26,14 +26,69 @@ const Footer = () => {
                     height={140}
                   />
                 </Link>
-                <p className="mb-9 text-base leading-relaxed text-body-color dark:text-body-color-dark">
-                  Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-                  Integer lobortis.
+                <p className="mb-4 text-base leading-relaxed text-body-color dark:text-body-color-dark">
+                  The Software Engineering (GL) branch at ENSIAS (École Nationale Supérieure d&apos;Informatique et d&apos;Analyse des Systèmes) is committed to producing highly skilled computer engineers.
                 </p>
-                <div className="flex items-center">
+                <li className="flex items-center mb-4">
+                  <svg
+                    className="w-8 h-8 mr-3 text-body-color dark:text-body-color-dark"
+                    fill="currentColor"
+                    xmlns="http://www.w3.org/2000/svg"
+                    viewBox="0 0 24 24"
+                  >
+                    <path d="M4 4h16c1.1 0 2 .9 2 2v12c0 1.1-.9 2-2 2H4c-1.1 0-2-.9-2-2V6c0-1.1.9-2 2-2zm0 2v12h16V6H4zm8 6l8-5H4l8 5zm0 2L4 9v8h16v-8l-8 5z" />
```

Comment:
this icon do not blend quietly with the other elements, and i think we can do better

Reply:
suggestion :
remove `<path> ... </path>`  node and replace it with

`<g xmlns="http://www.w3.org/2000/svg">
<polyline fill="none" points="4 8.2 12 14.1 20 8.2" stroke="#959cb1" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"></polyline>
<rect fill="none" height="14" rx="2" ry="2" stroke="#959cb1" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" width="18" x="3" y="6.5"></rect>
</g>`

sourced from : [https://www.svgrepo.com/svg/513836/mail](here)
license : CC